src/main.rs

```diff
@@ -31,9 +33,22 @@ fn main() {
             .long("namespace"))
         .get_matches();
     
-    let namespace = matches.value_of("namespace").unwrap();
+    let namespace_from_config = config::get();
+    let mut namespace = matches.value_of("namespace").unwrap();
+    if !namespace_from_config.is_empty() && namespace == "default" {
```

Comment:
what if the user specifies "default" as the namespace using "-n". which one should get more precedence? config file or -n argument?

Reply:
@dnivra26 I think -n argument should always have higher precedence. Let me fix that.

Reply:
@dnivra26 Fixed!